Add sentence about alert deactivation (#657)
conduktor/conduktor-docs: docs/platform/navigation/settings/alerts.md

- a destination: internal (Conduktor Console) or external (Slack, Teams or Webhook). Each alert can have **only one external destination**. To verify that it works, click **Send test**.

## Manage alerts
You can deactivate an alert temporarily without deleting it. The status of deactivated alerts will continue to update but will not be recorded in the alert's history, and notifications will not be fired.

> You can also test the alert at any point or duplicate it (e.g. if you want the same alert to go to different external destinations).

Author: how about

```suggestion
You can also test, duplicate or delete the alert at any point.
```
